ftsbot/cogs/antispam.py

```diff
@@ -40,6 +40,45 @@ async def on_message(
 		self,
 		message
 	):
+		if '@everyone' in message.content:
+			has_exception_role = False
+			for role in message.author.roles:
+				if role.name in {
+					'Discord Admins',
+					'Liquipedia Employee',
+					'Administrator'
+				}:
+					has_exception_role = True
+					break
+			if not has_exception_role:
+				# give muted role
+				mutedrole = discord.utils.get(message.guild.roles, name='Muted')
+				await message.author.add_roles(mutedrole)
+				# post message in staff channel
+				reporttarget = self.bot.get_channel(config.reporttarget)
+				time = message.created_at
+				await reporttarget.send(
+					embed=discord.Embed(
+						title=(
+							'Muted for potential (at)everyone spam - '
+							+ message.author.display_name + ' on ' + str(time)[:-7] + ' UTC:'
```

Comment:
```suggestion
							+ message.author.mention + ' on ' + str(time)[:-7] + ' UTC:'
``` I think this is better, what do you think @hjpalpha ?